commons/zenoh-config/src/lib.rs

```diff
@@ -1032,7 +1032,9 @@ impl<'a> serde::Deserialize<'a> for PluginsConfig {
 }
 impl std::fmt::Debug for PluginsConfig {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{:?}", &self.values)
+        let mut json = serde_json::to_value(self).unwrap();
```

Comment:
Why `unwrap()`? `fmt` returns a `Result`, proper error conversion should be made.

Reply:
I replicated the logic from: https://github.com/eclipse-zenoh/zenoh/blob/a67f9192d885d2af0d76ed64182944051d8130b7/commons/zenoh-config/src/lib.rs#L624-L630 

Will have to change that there as well then

Reply:
Yes please.